index.js

```diff
@@ -56,6 +56,11 @@ const loadURL = serve({ directory: `${__dirname}/dist` });
 
 	ipcMain.handle("getBucketName", async () => {
 		const { credentials } = await config.get();
+
+		if (!credentials) {
```

Comment:
Can we use `typeof credentials !== "object"`?

Reply:
Sure thing! :)